src/runtime_src/core/tools/xbreplay/src/main.cpp

```diff
@@ -0,0 +1,125 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright (C) 2024 Advanced Micro Devices, Inc. All rights reserved.
+
+#include "seq_reconstructor.hpp"
+#include "utils/cmd_args.hpp"
+#include "utils/logger.hpp"
```

Comment:
I like that you prefix with subdir `utils/` , but then modify the CMakeLists.txt to not have include search paths.

IMO, it is better to divide top CMakeLists into specific directory CMakeLists, aka one is this directory that build the executable from main.cpp and object library from utils. xbreplay_xrt, xbreplay_eng, etc.

Reply:
As discussed, since there are fewer files per sub folder, we will use single CMakeLists.